stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -435,15 +446,38 @@ public class MessageListViewModel(
         val messageMode = messageMode
 
         if (messageMode is MessageMode.MessageThread) {
-            threadMessagesState = threadMessagesState.copy(isLoadingMore = true)
-            chatDomain.threadLoadMore(channelId, messageMode.parentMessage.id, messageLimit)
-                .enqueue()
+            threadLoadMore(messageMode)
         } else {
             messagesState = messagesState.copy(isLoadingMore = true)
             chatClient.loadOlderMessages(channelId, messageLimit).enqueue()
         }
     }
 
+    /**
+     * Load older messages for the specified thread [MessageMode.MessageThread.parentMessage].
+     *
+     * @param threadMode Current thread mode.
+     */
+    private fun threadLoadMore(threadMode: MessageMode.MessageThread) {
+        threadMessagesState = threadMessagesState.copy(isLoadingMore = true)
+        if (ToggleService.isEnabled(ToggleService.TOGGLE_KEY_OFFLINE).not()) {
+            threadMessagesState = threadMessagesState.copy(isLoadingMore = true)
```

Comment:
I think we can remove this line. We are already changing this state two lines above